src/tests/customRender.tsx

```diff
@@ -1,27 +1,30 @@
-import React, { ReactElement } from 'react'
-import { render, RenderOptions } from '@testing-library/react'
+import { MemoryRouter } from 'react-router-dom';
+import { render, RenderOptions } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import CssBaseline from '@mui/material/CssBaseline';
+
 import CustomThemeProvider from '@/theme/CustomThemeProvider';
-import { MemoryRouter } from 'react-router-dom';
 import '@/i18n/config';
 
-// eslint-disable-next-line react-refresh/only-export-components
-const AllTheProviders = ({ children }: { children: React.ReactNode }) => {
-    return (
-        <CustomThemeProvider>
-            <CssBaseline />
-            <MemoryRouter>
-                {children}
-            </MemoryRouter>
-        </CustomThemeProvider >
-    )
-}
+export const customRender = (
+    ui: React.ReactElement,
+    options: Omit<RenderOptions, 'wrapper'> = {}
+) => {
+    const AllTheProviders = ({ children }: React.PropsWithChildren) => {
+        return (
+            <CustomThemeProvider>
+                <CssBaseline />
+                <MemoryRouter>{children}</MemoryRouter>
+            </CustomThemeProvider>
+        );
+    };
```

Comment:
moved this inside custom render so there is no need to do `// eslint-disable-next-line react-refresh/only-export-components`